docs/explanations/advanced.md

```diff
@@ -1,158 +1,178 @@
 
-You can read more about certificate issuers here: [certificate issuers documentation](../howtos/other/certificate_issuers.md)
+You can read more about certificate issuers at the [certificate issuers documentation](../howtos/other/certificate_issuers.md)
 
 ## Epinio installed components
 
-The official way to install Epinio is with the [Epinio helm chart](https://artifacthub.io/packages/helm/epinio/epinio).
-This helm chart installs Epinio and additional components, listed below, that are needed for Epinio to work.
+The official way to install Epinio is with the [Epinio Helm chart](https://artifacthub.io/packages/helm/epinio/epinio).
+This Helm chart installs Epinio and component dependencies, listed below, needed for Epinio to work.
 
 ### Epinio API server
 
-The main component of Epinio is the API server. The same `epinio` binary is both
-the server and the cli. The server is started with `epinio server` command in a properly configured Kubernetes Pod.
+The main component of Epinio is the API server.
+The same `epinio` binary is both the server and the CLI.
+The server starts with the `epinio server` command within a Kubernetes Pod, configured for Epinio.
 
-Epinio cli and web UI functionality are both implemented using the endpoints provided by the Epinio API server
-component. For example, when the user asks Epinio to "push" an application, the
-cli will contact the "Upload", "Stage" and "Deploy" endpoints of the Epinio API to upload the application code,
-create a container image for the application using this code and run the application on the cluster.
+Epinio CLI and web UI functionality are implemented using the endpoints provided by the Epinio API server component.
+For example, when the user asks Epinio to "push" an application,
+the CLI contacts the "Upload", "Stage" and "Deploy" endpoints of the Epinio API to:
+- upload the application code
+- create a container image for the application using this code
+- run the application on the cluster.
 
-The Epinio API server is running on the cluster and made accessible using Kubernetes resources like
+The Epinio API server runs on the cluster and is reached using Kubernetes resources like
 Deployments, Services,  Ingresses and Secrets.
 
 ### Kubed
 
-[Upstream documentation](https://github.com/kubeops/kubed)
+[Kubed documentation](https://github.com/kubeops/kubed)
 
-Kubed is installed by Epinio to keep secrets that are needed in more than
-one namespace synced. For example, the image pull secret is needed in every application namespace
+Epinio installs Kubed to keep secrets, needed in more than one namespace, synced.
+For example, the image pull secret is needed in every application namespace
 so that Kubernetes can pull the built application images from the [Container Registry](#container-registry).
 
-Kubed makes sure that if the source secret changes, the copy will change too.
+Kubed makes sure that if the source secret changes, the copies change too.
+
+<!--TODO: Still the case?-->
```

Comment:
AFAIK yes. @enrichman ?

Reply:
Hi @enrichman , Just to be clear i@m asking about whether it's still the case that 'Secret rotation' will be handled by Epinio in the future. Thanks,

Reply:
I'm not sure to understand which is the secrets we are referring to, and what kind of rotation. Regarding Kubed and the registry that's probably already the case. For other secrets I don't think that we are going to manage rotations somehow.